php/CacheBuster/CacheBuster.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+
+namespace Base\CacheBuster;
+
+
+class CacheBuster {
+
+    /**
+     * @var string
+     */
+    private $version = '';
+
+    const ENVIRONMENT_VARIABLE_NAME = 'BASE_CACHE_VERSION';
+
+    public function __construct() {
+
+        $environmentVersion = getenv(self::ENVIRONMENT_VARIABLE_NAME);
+        if($environmentVersion && $environmentVersion !== '') {
+            $this->version = $environmentVersion;
+            return;
+        }
+
+        if (defined('WP_ENV') && WP_ENV === 'development') {
+            // if in development mode then make each request get the file from the server
+            $this->version = (new TimestampVersionIdentifier())->getVersion();
+            return;
+        }
+    }
+
+    /**
+     * @return string
+     * @throws VersionUnidentifiableException
+     */
+    public function getVersion()
```

Comment:
Not sure what your point is here, are you suggesting the name of the class is wrong or the object shouldn't exist?